src/implementations/twig/components/form-group/form-group.html.twig

```diff
@@ -58,11 +73,21 @@
     disabled: true,
   }) %}
 {% endif %}
+
 {% if _invalid %}
   {% set _input = _input|merge({
     invalid: true,
   }) %}
+  {% if _id %}
```

Comment:
this is not clear to me, but it might just be the consequence of defining the group_id as an inner property of the input, here they are basically the result of a duplication, indeed to me this parameter should only be id and we don't need group_id at all, in this case.